compiler/IREmitter/Intrinsics/intrinsic-report.md

```diff
@@ -1564,4 +1564,4 @@
 
 ## Stats
 * Total:   1488
-* Visible: 238
```

Comment:
Note that the bump by 2 here is due to weirdness in previous merges. The new value is correct.